packages/core/src/utils/transformers/getArrayValue.ts

```diff
@@ -26,7 +26,24 @@ export function getArrayValue(
     if (typeof value === 'string') {
       return value;
     } else if (value?.sys?.type === 'Link') {
-      return entityStore.getEntityFromLink(value);
+      const resolvedEntity = entityStore.getEntityFromLink(value);
+      if (!resolvedEntity) {
+        return;
+      }
+      //resolve any embedded links - we currently only support 2 levels deep
+      const fieldKeys = Object.keys(resolvedEntity.fields || {});
+      fieldKeys.forEach((fieldKey) => {
+        const field = resolvedEntity.fields[fieldKey];
```

Comment:
Using `Object.entries` might be nice here since the loop is using both the `fieldKey` and `field`

```suggestion
      const fields = resolvedEntity.fields || {};
      Object.entries(fields).forEach(([fieldKey, field]) => {
```